examples/serve-zone.rs

```diff
@@ -2,10 +2,9 @@
 //!
 //! Try queries such as:
 //!
-//!   dig @127.0.0.1 -p 8053 NS example.com
-//!   dig @127.0.0.1 -p 8053 A example.com
-//!   dig @127.0.0.1 -p 8053 AAAA example.com
-//!   dig @127.0.0.1 -p 8053 CNAME example.com
+//!   dig @127.0.0.1 -p 8053 NS example.com dig @127.0.0.1 -p 8053 A
+//!   example.com dig @127.0.0.1 -p 8053 AAAA example.com dig @127.0.0.1 -p
+//!   8053 CNAME example.com
```

Comment:
This reformatting seems to have gone wrong somehow. Maybe we should put this in code fences instead of indenting?